### ISSUE TITLE
Sometimes getting a warning even if the npm module is installed.

### ISSUE THREAD

**Issue by ziedmahdi** [Meteor-Community-Packages/check-npm-versions#11]
I'm using typeahead from `corejs-typeahead`.

I added the above code in my package file:

<pre>
import {checkNpmVersions} from 'meteor/tmeasday:check-npm-versions';

checkNpmVersions({
  'corejs-typeahead': '0.11.x'
}, 'test:test');
</pre>

I'm getting this warning:

<pre>
WARNING: npm peer requirements (for test:test) not installed:
 - corejs-typeahead@0.11.x not installed.

Read more about installing npm peer dependencies:
  http://guide.meteor.com/using-packages.html#peer-npm-dependencies
</pre>

When I added code to require the package.json file the warning is gone.

<pre>
var z = require("/node_modules/corejs-typeahead/package.json");
console.log(z);
</pre>

![image](https://cloud.githubusercontent.com/assets/5032634/19385666/ba0bfcc0-9207-11e6-90c2-399566cb6c78.png)

[This is the duplication repo](https://github.com/ziedmahdi/npm-check-bug) 

Thank you very much.

**Comment by jakelacey2012** [comment]
I Believe I'm having the same issue also.

The only difference I have is I'm running my package in test mode.

```javascript
checkNpmVersions({
  'react-input-autosize': '1.1.x',
  'react-portal': '2.2.x',
}, 'lib-selectbox');
```

When running my package test everything is fine except I get these errors
```
I20161116-09:45:48.662(0)? [Error: Can't find npm module 'react-input-autosize/package.json'. Did you forget to call 'Npm.depends' in package.js within the 'modules-runtime' package?]
I20161116-09:45:48.851(0)? [Error: Can't find npm module 'react-portal/package.json'. Did you forget to call 'Npm.depends' in package.js within the 'modules-runtime' package?]
...
...
..
W20161116-09:45:48.864(0)? (STDERR) WARNING: npm peer requirements (for lib-selectbox) not installed:
W20161116-09:45:48.865(0)? (STDERR)  - react-input-autosize@1.1.0 not installed.
W20161116-09:45:48.865(0)? (STDERR)  - react-portal@2.2.1 not installed.
W20161116-09:45:48.865(0)? (STDERR) 
W20161116-09:45:48.866(0)? (STDERR) Read more about installing npm peer dependencies:
W20161116-09:45:48.866(0)? (STDERR)   http://guide.meteor.com/using-packages.html#peer-npm-dependencies
W20161116-09:45:48.866(0)? (STDERR) 
```

I am able to import these modules.

If I add this code below everything works fine and the errors are removed
```
require('react-input-autosize/package.json').version;
require('react-portal/package.json').version;
```

@ziedmahdi did you manage to fix this problem?

Thanks

**Comment by ziedmahdi** [comment]
In my case I get warnings and not errors.

**Comment by mcissel** [comment]
On the client side, the dependencies can't just be in package.json, they have to be imported into your app build.

I was using SSRWPO, which uses this package to ensure its dependencies are met. I was getting false positives on the client side only. To fix it, I had to import the dependencies into my project. I'm sure the warnings would go away once I finish integrating SSRWPO into the project.

**Comment by derwaldgeist** [comment]
Same here. But only for packages that haven't been required yet, as stated by @mcissel.

**Comment by kwunyeung** [comment]
Would this be fixed?

**Comment by theosp** [comment]
https://github.com/meteor/meteor/issues/9235

**Comment by copleykj** [comment]
I see no reason to run this on the client. The check can be run in server code and reported to the server console.
